packages/utils/__tests__/recursive-parser.test.tx

```diff
@@ -0,0 +1,64 @@
+"use strict";
+
+import defineTest from "../defineTest";
+
+defineTest(__dirname, "init", "fixture-1", "entry", {
```

Comment:
Have you tried to extend the types of defineTest ?

Reply:
Trying to expand the current types so the function is able to accept also other primitives.

Reply:
that helped passing test-cases, but don't we need to chagne the description as well. Param says `action`, and we are passing an `object`.
`action init, update or remove, decides how to format the AST` this is what it says.

Reply:
This func takes a name and extracts a testcase out of the params it get's. Code should be readable, and that is correct @hemal7735

Reply:
Okay, so I will go ahead and add more types to pass the test cases.
But we will need an issue to track if we really want to refactor this function?